js/controllers.js

```diff
@@ -215,9 +215,17 @@ appControllers.controller('HomeController', ['$scope', '$http', '$location', fun
           error: $scope.ajaxError
     });
   }
-
+  
   if(data["Admin"]) {
     $scope.user.type.push("Admin");
+    $scope.widgets = [
+    {
+      image: "/images/wireframe/image.png",
```

Comment:
The browser can't find this image. Is that just a placeholder? If so, plz put a TODO.
Otherwise, is the path correct? Did you upload the image?